tech/tech/interfaces/schemas/product_schema.py

```
@@ -0,0 +1,15 @@
+class ProductSchema(BaseModel):
+    name: str
+    price: float
+    category: Literal['Lanche', 'Acompanhamento', 'Bebida', 'Sobremesa']
+
+
+class ProductPublic(BaseModel):
+    id: int
+    name: str
+    price: float
+    category: str
+    model_config = ConfigDict(from_attributes=True)
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)